### PR TITLE
feat(pattern/checkbox): add label to checkbox

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [X] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [X] No

## Describe the changes

Github issue number or Jira issue URL:

### CODE REVIEW

File: src/docs/Components/Checkbox/code.mdx [hunk 1]
```diff
@@ -16,11 +16,14 @@ Import the settings and the checkbox `scss` files.
 ```
 
 ## Basic usage
+To create a classic checkbox you have 2 main class to apply:
```
Comment (nherwegh) [inline src/docs/Components/Checkbox/code.mdx]: (…) main class**ES**

File: src/docs/Components/Checkbox/code.mdx [hunk 2]
```diff
@@ -16,11 +16,15 @@ Import the settings and the checkbox `scss` files.
...
-To create a classic checkbox you have one main class to apply `mc-checkbox`.
+- `mc-checkbox` on div
+- `mc-checkbox__input` on input
+- `mc-checkbox__label` on label
 
 ```html
-<input type="checkbox" class="mc-checkbox" name="example" />
+<input id="checkbox-01" type="checkbox" class="mc-checkbox" name="example" />
+<label for="checkbox-01" class="mc-checkbox__label"> Label </label>
 ```
```
Comment (tiloyi) [inline src/docs/Components/Checkbox/code.mdx]: The code does not correspond to the text above. I think it should be: 

```HTML
<div class="mc-checkbox">
  <input id="checkbox-01" type="checkbox" class="mc-checkbox__input" name="example" />
  <label for="checkbox-01" class="mc-checkbox__label"> Label </label>
</div>
```

File: src/docs/Components/Checkbox/previews/intro.preview.html [hunk 3]
```diff
@@ -1,40 +1,48 @@
...
-      id="checkbox-01"
-      type="checkbox"
-      class="mc-checkbox"
-      name="example"
-      checked
-    />
-    <span class="example__label">checked</span>
+    <div class="mc-checkbox">
+      <input
+        id="checkbox-01"
```
Comment (tiloyi) [inline src/docs/Components/Checkbox/previews/intro.preview.html]: ~~id="checkbox-01"~~ => id="checkbox-02"

File: src/docs/Components/Checkbox/previews/intro.preview.html [hunk 4]
```diff
@@ -1,40 +1,48 @@
...
+        type="checkbox"
+        class="mc-checkbox__input"
+        name="example"
+        checked
+      />
+      <label for="checkbox-01" class="mc-checkbox__label">checked</label>
```
Comment (tiloyi) [inline src/docs/Components/Checkbox/previews/intro.preview.html]: ~~for="checkbox-01"~~ => for="checkbox-02"

File: src/docs/Components/Checkbox/previews/intro.preview.html [hunk 5]
```diff
@@ -1,40 +1,48 @@
...
-      id="checkbox-01"
-      type="checkbox"
-      class="mc-checkbox"
-      name="example"
-      disabled
-    />
-    <span class="example__label">disabled</span>
+    <div class="mc-checkbox">
+      <input
+        id="checkbox-01"
```
Comment (tiloyi) [inline src/docs/Components/Checkbox/previews/intro.preview.html]: ~~id="checkbox-01"~~ => id="checkbox-03"
~~for="checkbox-01"~~ => for="checkbox-03"

File: src/docs/Components/Checkbox/previews/intro.preview.html [hunk 6]
```diff
@@ -1,40 +1,48 @@
...
-    <input
-      id="checkbox-01"
-      type="checkbox"
-      class="mc-checkbox is-invalid"
-      name="example"
-    />
-    <span class="example__label">invalid</span>
+    <div class="mc-checkbox">
+      <input
+        id="checkbox-01"
```
Comment (tiloyi) [inline src/docs/Components/Checkbox/previews/intro.preview.html]: ~~id="checkbox-01"~~ => id="checkbox-04"
~~for="checkbox-01"~~ => for="checkbox-04"

File: src/docs/Components/Checkbox/code.mdx (hunk 2 above)
Comment (mohamedMok) [reply src/docs/Components/Checkbox/code.mdx]: Good catch! 😎

**Pull request merged**